omnigibson/systems/system_base.py

```diff
@@ -1109,32 +1112,54 @@ def _customize_particle_material(mat: MaterialPrim): --> None
         system_type = metadata["type"]
         system_kwargs = dict(name=system_name)
 
-        def generate_particle_template_fcn():
-            return lambda prim_path, name: \
-                og.objects.DatasetObject(
-                    prim_path=prim_path,
-                    name=name,
-                    usd_path=os.path.join(system_dir, f"{system_name}.usd"),
-                    category=system_name,
-                    visible=False,
-                    fixed_base=False,
-                    visual_only=True,
-                    include_default_states=False,
-                    abilities={},
-                )
+        asset_path = os.path.join(system_dir, f"{system_name}.usd")
+        has_asset = os.path.exists(asset_path)
+        if not has_asset:
+            if system_type == "macro_visual_particle":
+                # Fallback to stain asset
+                asset_path = os.path.join(gm.ASSET_PATH, "models", "stain", "stain.usd")
+                has_asset = True
```

Comment:
Let's put the stain particle to be under `og_dataset/objects`?

Reply:
Yep, sg! I think we should only make this change once we have a finalized dataset to be released